frameworks/cassandra/universe/marathon.json.mustache

```diff
@@ -266,7 +267,10 @@
     "TASKCFG_ALL_CASSANDRA_ENABLE_USER_DEFINED_FUNCTIONS": "{{cassandra.enable_user_defined_functions}}",
     "TASKCFG_ALL_CASSANDRA_ENABLE_SCRIPTED_USER_DEFINED_FUNCTIONS": "{{cassandra.enable_scripted_user_defined_functions}}",
 
-    "TASKCFG_ALL_METRICS_PERIOD_MIN": "{{cassandra.metrics_period_min}}",
+    {{#cassandra.prometheus_exporter.prometheus_exporter_enabled}}
+    "TASKCFG_ALL_PROMETHEUS_EXPORTER_ENABLED": "{{cassandra.prometheus_exporter.prometheus_exporter_enabled}}",
+    {{/cassandra.prometheus_exporter.prometheus_exporter_enabled}}
+    "TASKCFG_ALL_PROMETHEUS_EXPORTER_CONFIGURATION_OPTIONS": "{{cassandra.prometheus_exporter.configuration_options}}",
```

Comment:
Do we need this `TASKCFG_ALL_` prefix? Since we don't need them in any configuration file.